CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+`pwru` is an open source project. The userspace code is licensed under [Apache-2.0](LICENSE), while the BPF under [BSD 2-Clause](bpf/LICENSE.BSD-2-Clause) and [GPL-2.0](bpf/LICENSE.GPL-2.0). Everybody is welcome to contribute. Contributors are required to follow the [CNCF Code of
+Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+## Getting Started
+
+- Read the [README](https://github.com/cilium/pwru#readme) to understand how `pwru` works and its use cases.
+- Check out the [issues list](https://github.com/cilium/pwru/issues) for open tasks, especially those labeled `good first issue`.
+
+## Developing  
```

Comment:
Could we keep this section in the README.md, and then reference from CONTRIBUTING.md? I'm afraid that many might not look into the CONTRIBUTING.md to find those details.

Reply:
which section do you mean exactly? The developing section?

Reply:
Yep, the `##  Developing`. Could we keep it in the README.md?